nixos/modules/security/audit.nix

```diff
@@ -16,7 +16,6 @@ let
   # The order of the fixed rules is determined by augenrules(8)
   rules = pkgs.writeTextDir "audit.rules" ''
     -D
-    -b ${toString cfg.backlogLimit}
```

Comment:
We should probably keep this flag to allow changing the backlog limit using `nixos-rebuild switch` at runtime. Probably not a super big deal to have it more rigid, but runtime changes are probably nice to have.